tests/main/interfaces-fuse_support/task.yaml

```diff
@@ -28,7 +28,12 @@ prepare: |
     mkdir -p $MOUNT_POINT
 
 restore: |
-    umount $MOUNT_POINT || true
+    # remove the mount point
+    mounts=$(nsenter --mount=/run/snapd/ns/test-snapd-fuse-consumer.mnt cat /proc/mounts | \
+             grep "$(basename $MOUNT_POINT) fuse" | cut -f2 -d' ')
+    for m in $mounts; do
+        nsenter --mount=/run/snapd/ns/test-snapd-fuse-consumer.mnt umount "$m"
+    done
```

Comment:
I wonder if we should run the `mount=...` command again after the umount to see if things are really cleaned up?

Reply:
Wonder if it's needed, none of the lazy flags are used, so if umount fails (device busy or similar) we'll know it because `restore` will fail as well.